src/pages/Comment/Components/CommentHeader.js

```diff
@@ -53,8 +49,7 @@ function CommentHeader(props) {
           </SortMenu>
         </HeaderMenu>
         <div>
-          {getRedStars(ratingAvg)}
-          {getEmptyStars(ratingAvg)}
+          {getStars(ratingAvg)}
```

Comment:
이런 형태 보다는 Rating 자체를 컴포넌트화 해서 사용하게끔 만드시는게 더 깔끔할 것 같습니다.

```suggestion
<Rating 
  ratingValue={ratingAvg}
/>
```